_includes/markdown/React.md

```diff
@@ -151,3 +151,18 @@ The presence of Element is why you don't see React directly imported into Gutenb
 Gutenberg offers a library of higher-order components (HOC) you can use to build out a robust editor experience. The features of these components range from focus management to auditory messaging. It is best to familiarize yourself with these components so you don't end up rebuilding a utility functionality that already exists within Gutenberg. You can view [Gutenberg's library of generic Higher Order React Components](https://github.com/WordPress/gutenberg/tree/master/packages/components/src/higher-order) to learn more or view the official [React documentation for general information about using HOC](https://reactjs.org/docs/higher-order-components.html).
 
 As with any evolving feature, it is important to frequently check the documentation for new additions and updates.
+
+### Whitelisting / Blacklisting
+
+Gutenberg has the ability to whitelist or blacklist certain blocks. This can be found under the [Block Filters](https://developer.wordpress.org/block-editor/developers/filters/block-filters/) page of the official Gutenberg documentation. Specifically, under the heading [Removing Blocks](https://developer.wordpress.org/block-editor/developers/filters/block-filters/#removing-blocks). Details of how to filter via JavaScript or PHP can be found there.
+
+Also note that this method is different than the `allowedBlocks` prop on `<InnerBlocks>` when used in a custom block. Whitelisting or blacklisting here applies to all blocks on a higher level within the editor.
+
+There may be a need to remove or restrict access to blocks. This could be for a variety of reasons, but typical scenarios could be:
```

Comment:
@Firestorm980 can we flesh this recommendation out a bit more? Maybe something more around "this is when you whitelist" vs. "this is when you blacklist"?

Reply:
Updated